.github/workflows/ci.yml

```diff
@@ -315,7 +320,17 @@ jobs:
       - name: Run smoke-test
         timeout-minutes: 5
         run: |
-          make RELEASE=true UNINSTALL=noclean smoke-test-tls-root
+          FLV_CMD=true FLV_SOCKET_WAIT=600 make RELEASE=true UNINSTALL=noclean smoke-test-tls-root
```

Comment:
Can you move environment variable into ENV section at the beginning of job? Can look at other job as example: